pkg/agent/params.go

```diff
@@ -60,9 +60,9 @@ func getParameters(config *NodeBootstrappingConfiguration, generatorCode string,
 
 	// Kubernetes Parameters
 	if properties.OrchestratorProfile.IsKubernetes() {
-		assignKubernetesParameters(properties, parametersMap, cloudSpecConfig, generatorCode)
+		assignKubernetesParameters(properties, parametersMap, *cloudSpecConfig, generatorCode)
```

Comment:
nit: I'd change signature of assignKubernetesParameters to take in a pointer

Reply:
Fixed to take a pointer as param.